src/datasources/db/v2/__tests__/querybuilder.mock.ts

```diff
@@ -0,0 +1,6 @@
+export const mockQueryBuilder = {
+  delete: jest.fn().mockReturnThis(),
+  from: jest.fn().mockReturnThis(),
+  where: jest.fn().mockReturnThis(),
+  execute: jest.fn(),
+};
```

Comment:
Nit: let's type this to keep maintain type safety.

Reply:
Done https://github.com/safe-global/safe-client-gateway/pull/2084/commits/d879b9176acbbdeeb93f2a51942c5afc52149ec4